site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -203,313 +206,200 @@ export const CreateWorkspacePageView: FC<
-                <p className={styles.formSectionInfoDescription}>
-                  The user that is going to own this workspace. If you are
-                  admin, you can create workspace for others.
-                </p>
-              </div>
+            </FormFields>
+          </FormSection>
+        )}
 
-              <Stack
-                direction="column"
-                spacing={1}
-                className={styles.formSectionFields}
-              >
-                <UserAutocomplete
-                  value={props.owner}
-                  onChange={props.setOwner}
-                  label={t("ownerLabel")}
+        {/* Template git auth */}
+        {props.templateGitAuth && props.templateGitAuth.length > 0 && (
+          <FormSection
+            title="Git Authentication"
+            description="This template requires authentication to automatically perform
+            Git operations on create."
+          >
+            <FormFields>
+              {props.templateGitAuth.map((auth, index) => (
+                <GitAuth
+                  key={index}
+                  authenticateURL={auth.authenticate_url}
+                  authenticated={auth.authenticated}
+                  type={auth.type}
+                  error={gitAuthErrors[auth.id]}
                 />
-              </Stack>
-            </div>
-          )}
-
-          {/* Template git auth */}
-          {props.templateGitAuth && props.templateGitAuth.length > 0 && (
-            <div className={styles.formSection}>
-              <div className={styles.formSectionInfo}>
-                <h2 className={styles.formSectionInfoTitle}>
-                  Git Authentication
-                </h2>
-                <p className={styles.formSectionInfoDescription}>
-                  This template requires authentication to automatically perform
-                  Git operations on create.
-                </p>
-              </div>
+              ))}
+            </FormFields>
+          </FormSection>
+        )}
 
-              <Stack
-                direction="column"
-                spacing={2}
-                className={styles.formSectionFields}
-              >
-                {props.templateGitAuth.map((auth, index) => (
-                  <GitAuth
-                    key={index}
-                    authenticateURL={auth.authenticate_url}
-                    authenticated={auth.authenticated}
-                    type={auth.type}
-                    error={gitAuthErrors[auth.id]}
+        {/* Template params */}
+        {props.templateSchema && props.templateSchema.length > 0 && (
+          <FormSection
+            title="Template params"
+            description="Those values are provided by your template's Terraform
+            configuration."
+          >
+            <FormFields>
+              {props.templateSchema
+                // We only want to show schema that have redisplay_value equals true
```

Comment:
I like the addition of a comment here, but I'm still unclear as to why we don't want to show schemas with `redisplay_value=false`